libp2p/security/insecure_security.py

```diff
@@ -0,0 +1,44 @@
+from libp2p.security.secure_transport_interface import ISecureTransport
+from libp2p.security.secure_conn_interface import ISecureConn
+
+class InsecureTransport(ISecureTransport):
```

Comment:
Is the InsecureTransport/InsecureConn needed or is it just for testing? I felt as though if `sec_opt` is `None` then we should just avoid anything to do with security instead of using something like an InsecureTransport. At least, it might be confusing for this to be a subtype of ISecureTransport.

Reply:
This follows the precedent set forth by go. The idea is that you _must_ choose a security module, but you are allowed to choose one that doesn't do anything.